scripts/update-drafts.ts

```diff
@@ -35,9 +35,14 @@ async function main() {
   const compat = new Compat();
 
   // Build a set of used BCD keys.
-  const usedFeatures = new Set<string>(
-    Object.values(features).flatMap((data) => data.compat_features)
-  );
+  const webFeatures = new Map<string,string>();
```

Comment:
I'd like some other name than `webFeatures` since that sounds like the project as a whole and I'd guess they keys are web-features identifiers. This is a reversed map of web-features. How about `compatFeatureToName`, also making it more obvious that it's the feature name, not ID, being used here.